packages/template-retail-react-app/package.json

```diff
@@ -58,7 +58,7 @@
     "@testing-library/user-event": "^14.4.3",
     "babel-plugin-module-resolver": "5.0.2",
     "base64-arraybuffer": "^0.2.0",
-    "bundlesize2": "^0.0.31",
+    "bundlesize2": "^0.0.35",
```

Comment:
I don't think we are using this dependency for anything, so it might be ok to just delete?

Reply:
It is! we use it to check bundlesize